tf_mat_world_flan1:
  target_pose:
    position: [0.1417057, 0.5319608, 0.09579612]
    orientation:
      - [0.9967777, 0.07060843, 0.03806327]
      - [0.07062963, -0.9975023, 0.0007890977]
      - [0.03802392, 0.00190184, -0.999275]
tf_mat_world_flan4:
  target_pose:
    position: [0.127534, -0.5316753, 0.09552621]
    orientation:
      - [1, 2.601523e-06, -1.294753e-06]
      - [2.601524e-06, -1, 8.191149e-07]
      - [-1.294751e-06, -8.191183e-07, -1]
tf_mat_base_link1_0:
  target_pose:
    position: [0.072125, 0.072125, 0.018]
    orientation:
      - [-2.597348e-06, -0.7071068, 0.7071068]
      - [-2.597348e-06, -0.7071068, -0.7071068]
      - [1, -3.673205e-06, -2.220446e-16]
tf_mat_base_link4_0:
  target_pose:
    position: [0.072125, -0.072125, 0.018]
    orientation:
      - [-2.597348e-06, -0.7071068, -0.7071068]
      - [2.597348e-06, 0.7071068, -0.7071068]
      - [1, -3.673205e-06, 2.220446e-16]
tf_mat_base_link2_0:
  target_pose:
    position: [-0.072125, 0.072125, 0.018]
    orientation:
      - [2.597348e-06, 0.7071068, 0.7071068]
      - [-2.597348e-06, -0.7071068, 0.7071068]
      - [1, -3.673205e-06, 5.551115e-16]
tf_mat_base_link3_0:
  target_pose:
    position: [-0.072125, -0.072125, 0.018]
    orientation:
      - [2.597348e-06, 0.7071068, -0.7071068]
      - [2.597348e-06, 0.7071068, 0.7071068]
      - [1, -3.673205e-06, -2.220446e-16]